doc/adr/0002-use-meta-yaml-to-track-feedstock-metadata.md

```diff
@@ -31,13 +31,29 @@ The format and contents of this file are specified as follows.
 ### Top Level Data
 
 ```yaml
-id: noaa-oisst  # top-level ID for the feedstock. must be unique and must match feedstock repo name
+id: noaa-oisst
+version: "1.0" 
 title: "NOAA Optimum Interpolated SST"
 description: "Analysis-ready Zarr datasets derived from NOAA OISST NetCDF"
 pangeo_forge_recipes_version: "0.1"
-pangeo_forge_metadata_spec_version: "1"
+pangeo_forge_metadata_spec_version: "2"
 ```
 
+| name | description | valid characters |
+| -- | -- | -- |
+| id | top-level ID for the feedstock. must be unique and must match feedstock repo name | lower case ascii alpha-numeric, no spaces, only dashes allowed |
+| version | feedstock version | `{MAJOR}.{̨MINOR}` |
+| description | - | Unicode |
+
+#### Version
+
+We use a two-element version of semantic versioning: `MAJOR.MINOR`.
+The recipe version starts at 1.0.
+Updates to the minor version number are considered minor changes;
+when a new minor version is released, bakeries will _overwrite the previous minor version_ of the feedstock's datasets.
+Instead, the bakery should _always retain the latest major version_.
+Updates that involve concatenation of new elements (e.g. extend the temporal range) are considered minor version updates.
```

Comment:
> didn't you just say that this would overwrite the dataset?

Yes I see how that is confusing. What I mean is that 3.6 ➡️ 3.7 will make 3.6 go away. We won't keep a separate copy of 3.6 around, since 3.7 supersedes it. But this may mean just renaming 3.6 to 3.7 and then appending to it. So I agree that "overwrite" is not necessarily the right word here.

> if 3.7 was followed by a 4.x series, then 3.7 will be kept for good, archival?

Correct.